doc/MAGIC.md

```diff
@@ -831,12 +831,12 @@ Character status value  | Description
 `SOCIAL_LIE`            | Affects your ability to lie.
 `SOCIAL_PERSUADE`       | Affects your ability to persuade.
 `READING_EXP`           | Changes the minimum you learn from each reading increment.
-`RECOIL_MODIFIER`       | Affects recoil when shooting a gun. Can be applied only to guns. Positive value increase the dispersion, negative decrease one.
+`RECOIL_MODIFIER`       | Affects recoil when shooting a gun. Positive value increase the dispersion, negative decrease one.
```

Comment:
Are the RECOIL_MODIFIER changes your changes or an old version applying again?

Reply:
Old verison - i thought it works only with guns, but then it was revealed it just didn't work because of flexbuffer error, and after fix it indeed started to work with any item